src/index.js

```diff
@@ -1,13 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
+import Calculator from './components/calculator';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
```

Comment:
- [ ] It is the `App.js` that is supposed to be imported inside your index.js. `App.js` should be the parent for all your components in your application and `index.js` would now be the parent to `App.js`. Kinldy look into that and fix.